src/main/scala/viper/gobra/frontend/Parser.scala

```diff
@@ -1100,6 +1105,15 @@ object Parser {
       case left ~ right => PMultisetConversion(PRangeSequence(left, right).range(left, right))
     }
 
+    lazy val optionNone : Parser[POptionNone] =
+      "none" ~> ("[" ~> typ <~ "]") ^^ POptionNone
+
+    lazy val optionSome : Parser[POptionSome] =
+      "some" ~> ("(" ~> expression <~ ")") ^^ POptionSome
+
+    lazy val optionGet : Parser[POptionGet] =
+      "option" ~> ("(" ~> expression <~ ")") ^^ POptionGet
```

Comment:
No, you are right in that "get" is much more standard and intuitive.

However, I didn't want to make "get" a reserved keyword, since it is probably used a lot in 'real-world' Go programs. So I opted for "option" instead, since it already is a reserved keyword.

Do you think it is better to use "get"?

Reply:
Some langauges use "the", I believe.

Reply:
I agree that it could be pretty annoying to have `get` as a reserved keyword. Do we have a general strategy so far? Simply don't care and force programmers to respect all reserved keywords or go for some special prefix to decrease likelihood (or even better use an invalid prefix in go)?

Reply:
I don't think there is a general strategy at the moment.

One possibility is to make `gobra` a reserved keyword, and use something like `gobra:get`.

Reply:
That approach seems reasonable, if we ever need to add extra operations to the language we can always use the `gobra` keyword to prefix them

Reply:
I like this idea as well. Another extension of that would be to write a "fake" import that would allow you to specify the qualifier (`gobra`, `.` or any other identifier)

Reply:
Yes, I like the "fake import" idea.

Reply:
Me too.

Reply:
There is a small problem there. Since Go does not support polymorphism, so the import does really have to be fake, since the functions we provide could not be defined in Go.